tests/common/plugins/test_completeness/__init__.py

```diff
@@ -4,10 +4,37 @@
 import warnings
 
 class CompletenessLevel(enum.IntEnum):
-    debug = 0 # Minimum execution
-    basic = 1
-    confident = 2
-    thorough = 3 # Maximum execution
+    diagnose = 0
+    debug = 1 # Minimum execution
+    basic = 2
+    confident = 3
+    thorough = 4 # Maximum execution 
+
+    @classmethod
+    def get_normalized_level(cls, request):
+        """Takes a request instance and returns normalized completeness level in string format.
+        For example, if a testcase supports "CompletenessLevel.basic, CompletenessLevel.thorough", and specified level
+        during test execution is "confident", this method will make use of normalization logic during
+        pytest_runtest_setup and returns the normalized level as "basic" (str type of CompletenessLevel.basic)
+        """
```

Comment:
I have a few suggestions here:
1. Separate the summary line and the example
2. Generalize the description a little bit (i.e. describe *what* is happening rather than *how* it is happening)
3. Explicitly call out the return type

One possible interpretation:
```suggestion
        """Get the normalized completeness level for a given test instance.
        
        For example, if a testcase supports "CompletenessLevel.basic, CompletenessLevel.thorough", and the specified level
        during test execution is "confident", then this method will normalize the level to "basic".
        
         Returns:
             CompletenessLevel as a string
        """
```

Reply:
Incorporated the suggestion. Thanks!